packages/expo-symbols/src/SymbolModule.types.ts

```diff
@@ -1,10 +1,11 @@
 import { ViewProps } from 'react-native';
+import { SFSymbol } from 'sf-symbols-typescript';
 
 export interface SymbolViewProps extends ViewProps {
```

Comment:
Not related to the changes in this PR, but we should prefer `types` over `interfaces` in general, especially when it comes to the component props.